mqtt/mqtt-edgehub/src/command.rs

```diff
@@ -75,11 +75,12 @@ impl CommandHandler {
     }
 
     pub async fn run(mut self) -> Result<(), CommandHandlerError> {
-        let subscribe_topics = &[TOPIC_FILTER.to_string()];
+        debug!("starting command handler");
 
+        let subscribe_topics = &[TOPIC_FILTER.to_string()];
         if !self.subscribe(subscribe_topics).await? {
             return Err(CommandHandlerError::MissingSubacks(
-                subscribe_topics.concat(),
+                subscribe_topics.to_vec().join(", "),
```

Comment:
```suggestion
                subscribe_topics.join(", "),
```
nit: I believe you don't need `to_vec()`

Reply:
But from a design point of view, I'd rather pass array itself. Later on we may want to iterate through a list of topics for some reason. In the log it also be nice JSON like array
```suggestion
                return Err(CommandHandlerError::MissingSubacks(*subscribe_topics);
```

Reply:
So if we have ```*subscribe_topics``` then it is ```[std::string::String; 1]```. How do we get the String out safely? We can use ```get()``` but will need to handle the error here which creates nested errors.

Reply:
Yeah, that's true...